pom.xml

```diff
@@ -274,6 +274,8 @@
           https://maven.apache.org/surefire/maven-surefire-plugin/examples/class-loading.html -->
           <useSystemClassLoader>true</useSystemClassLoader>
           <useManifestOnlyJar>false</useManifestOnlyJar>
+          <forkCount>4</forkCount>
```

Comment:
do we have to hard code a value here?

Reply:
> do we have to hard code a value here?

We have to specify a value but according to documentation, it can be related to the number of processor on the machine: 
> if you terminate the value with a 'C', that value will be multiplied with the number of available CPU cores in your system. For example forkCount=2.5C on a Quad-Core system will result in forking up to ten concurrent JVM processes that execute tests.